scripts/unsupported/combine_tracks_postprocessing_cnv/combine_tracks.wdl

```diff
@@ -0,0 +1,621 @@
+# A postprocessing workflow for GATK CNV.
+#
+# THIS CANNOT BE RUN IN TUMOR-ONLY MODE.  A MATCHED NORMAL IS REQUIRED.
+#
+# Internally, we have shown that this workflow improves sensitivity and precision.  Additionally, this workflow is quite
+#  cheap to run.  Note that auxiliary files are required and without these exact files, performance may suffer.
+#
+# UNSUPPORTED and we may modify this WDL before we fully support it.
+#
+# This will also generate absolute (https://software.broadinstitute.org/cancer/cga/absolute) compatible files.
+#  Currently, the balanced-segment calling is extremely ham-fisted.
+#
+# This workflow:
+#  - Does conversion of output seg files to a format compatible with IGV.  This will be removed soon, since the GATK tools
+#     have been updated to produce files that can be easily read by IGV.
+#  - Tags possible germline events that appear in the tumor results.  This is done by comparing breakpoints and
+#     reciprocal overlaps.  This algorithm takes into account any hypersegmentation.
+#  - Tags tumor events that appear in the given blacklists (centromere and gistic blacklist)
+#  - Removes any events that have been tagged in the previous two steps.  Any resulting gaps are covered, so long as
+#     the segment mean would remain constant and the distance is less than max_merge_distance (default 1Mb as of
+#     this writing)
+#
+#  Outputs:
+#  - seg file that is has germline events removed (and gaps merged).  Same columns as a called seg file.
+#      This should be considered the final output.  IGV compatible.  The results have been evaluated.
```

Comment:
I'd remove "The results have been evaluated." as it's not very informative.  Either that, or expand with more detail, preferably above.